docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -106,6 +113,28 @@ DELETE FROM                          all                delete
 UPDATE                               all                update
 ==================================== ========== ======= ==================== ===================================================
 
+Permissions required for executing functions:
+
+.. list-table::
+   :widths: 30, 10, 15, 15, 30
+   :header-rows: 1
+
+   * - SQL command
+     - Catalog
+     - Function permission
+     - Function kind
+     - Note
+   * - ``SELECT function()``
+     -
+     - ``execute``, ``grant_execute*``
+     - ``aggregate``, ``scalar``, ``window``
+     - ``grant_execute`` is required when function is executed with view owner privileges.
+   * - ``SELECT FROM TABLE(table_function())``
+     - ``all``
+     - ``execute``, ``grant_execute*``
+     - ``table``
+     - ``grant_execute`` is required when PTF is executed with view owner privileges.
```

Comment:
Is it well know what PTF means? I would prefer to use table function. Same in other places.

Reply:
are all table functions polymorphic? If not, we need to differentiate the two. Google style guide and numerous other documentation standards specify that you spell term out the first time it is used, with the acronym. Thereafter, you just use the acronym. When things have entered very common usage (like HTTP, unless it is actual documentation for that protocol), you don't have to spell it out.

Reply:
No .. PTF is not well known .. use "table functions" .. and @m57lyra .. not all table functions are polymorphic .. check out the video I created and shared recently

Reply:
Maybe make a link to https://trino.io/docs/current/functions/table.html in every occurrence of "PTF" ?

Reply:
I linked every `table function` to https://trino.io/docs/current/functions/table.html